### DISCUSSION TITLE
dm history reset

### DISCUSSION THREAD

**Discussion by helpmonmon** [drizzle-mizzle/CharacterAI-Discord-Bot#74] [Q&A]
hi! i got the latest release and dm'd one of my bots but it had started a new chat on its own. i checked on the c.ai website and indeed, it left the original conversation we were having and started a new chat. i'd like to go back to that one, but i'm not sure how to get the history id since it's in dms. i mostly use the bots in dms so.. yeah. i tried the get history command, but it gave me the history of the new chat it created.. how do i get back to the old chat? thank you in advance!

edit: i figured out that the history id can be found in the url too, but when i do it in dms, it says this:

Note: some commands are not intended to be called from DMs
:warning: Failed to execute command: Channel not found. (ObjectNotFound)

**Comment by drizzle-mizzle** [comment]
Yeah, it won't work with dm right now, sorry. I'll implement it later.

**Reply by helpmonmon** [reply]
fair enough, of course! is there any other way to revert to the old chat from dms or should i just chat on c.ai? sorry to keep asking, thank you for your reply :)

**Reply by drizzle-mizzle** [reply]
Well, you can find your dm chat on /storage/channels.yaml and change history id manually...

**Reply by helpmonmon** [reply]
didn't know that, cool, thank you :)

**Reply by drizzle-mizzle** [reply]
Forgot to mention it was fixed in last update
